client/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "ui-planeterrella"
+  }
+}
```

Comment:
Why are all of the changes in client? I thought we had discussed keeping it in the backend to be called by the api.